10.9-libcxx/stable/main/finkinfo/sci/fftw3.info

```diff
@@ -28,11 +29,7 @@ CompileScript: <<
  #!/bin/sh -ev
  gcclib=`%p/bin/gfortran-fsf-%type_raw[gcc] --print-lib`
  FLIBDIR=`dirname $gcclib`
- if [ x%m == x"powerpc" ]; then
```

Comment:
Hurrah for removing powerpc legacy cruft!

Reply:
👍 You're welcome!